code/obj/machinery/deep_fryer.dm

```diff
@@ -26,7 +26,7 @@
 			boutput(usr, "<span class='alert'>The [src] refuses to interface with you, as you are not a properly trained chef!</span>")
 			return
 		if (W.cant_drop) //For borg held items
-			user.show_text("You can't put that in [src] when it's attached to you!", "red")
+			boutput(user, "You can't put that in [src] when it's attached to you!")
```

Comment:
This (and the other kitchen stuff ones) aren't really related to this PR. The code you've replaced is also not comparable, as the previous stuff is "red" (so you'd want something like `boutput(user, "<span class='alert'>You can't put that in [src] when it's attached to you!</span>"). (See three lines below this one)

Personal take: split into a second PR. Lots of little, unrelated PRs is better than one PR with multiple unrelated changes.